docs/FAQ.md

```diff
@@ -128,6 +128,12 @@ Release information can be found here:
 Open Issues can be found here: 
 - [Open Issues](https://github.com/vmware/octant/issues)
 
+## Q: Can Octant connect to multiple clusters at the same time?
+
+No.
+
+Octant can only connect to a single cluster at a time. For convienece Octant provides a context switcher that allows you to select the current context Octant is using without having to restart Octant.
```

Comment:
Can the second sentence be re-worded so it doesn't use `Octant` three times? I'll approve after that.